### DISCUSSION TITLE
Possible to package .NET runtime?

### DISCUSSION THREAD

**Discussion by boydfields** [Hofknecht/SystemTrayMenu#385] [Q&A]
Hi. I am new to building from source. Is it possible to package the .NET runtime (i.e. the files required to run STM) along with the executable? I have done some searching and I believe it is possible but having tried I cannot make it work. The use-case is when moving from computer to computer using STM as a portable app. Saves having to install anything extra. Welcome to thoughts - Thank you!

**Comment by Hofknecht** [comment]
Thanks for the idea!
There is an article from windows on this here. 
https://docs.microsoft.com/de-de/dotnet/core/deploying/#publish-self-contained

That sounds very interesting, we have to try it!
